spec/factories/whitehall_export/index_factory.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :whitehall_export_index, class: Hash do
+    skip_create
+
+    documents do
+      [
+        build(:whitehall_export_index_document),
+      ]
+    end
```

Comment:
Minor: might want to put this on one line?

Reply:
Sure, updated to be on one line